### DISCUSSION TITLE
Icon Size for Lights Card

### DISCUSSION THREAD

**Discussion by jsp196** [home-assistant/frontend#8358] [Dashboard]
Would it be possible to add the icon size option for the Lights card in a similar way that it is available on the Button card?

**Comment by iantrich** [comment]
I honestly don't like that it's an option on the button card...

**Reply by jsp196** [reply]
If it's not an option, it should at least be functional in the code view.  I have one dashboard that is used with 8 large buttons and that's all it needs to be.  I want them as gigantic buttons so it's easy to use.  Right now it's not even functional on a Light card, afaik.

**Reply by spacegaier** [reply]
> 
> 
> I honestly don't like that it's an option on the button card...

Why not? Offers flexibility to create nice looking dashboards. The default size can be huge, compared to a manual value:

![image](https://user-images.githubusercontent.com/114137/108400308-59ba5800-721b-11eb-8cb8-50cb58e49016.png)

![image](https://user-images.githubusercontent.com/114137/108400466-7f476180-721b-11eb-9fe3-8ee2f591caf4.png)

But for light cards it is a bit more tricky since we have the icon and the round slider around it for dimmable lights. So just changing the icon does not really help.

![image](https://user-images.githubusercontent.com/114137/108400661-c46b9380-721b-11eb-89e5-6f5e36aca736.png)

![image](https://user-images.githubusercontent.com/114137/108400858-fb41a980-721b-11eb-88f8-e36d32c158ac.png)

**Reply by iantrich** [reply]
Round slider needs to go
